packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.web.tsx

```diff
@@ -0,0 +1,283 @@
+import * as React from "react"
+import {
+    Animated,
+    PanResponder,
+    StyleSheet,
+    GestureResponderEvent,
+    PanResponderGestureState,
+    View,
+    LayoutChangeEvent,
+    Pressable
+} from "react-native"
+
+import type  { UICarouselViewContainerProps, UICarouselViewPageProps } from "../../types";
+import { useAnimatedValue } from "../hooks";
+import { AnimationConfig } from "./constants";
```

Comment:
Oh, I see it now. You made it work on web using regular `Animated`...

To be honest I'd like to see `react-native-reanimated` here instead, especially because it works great on web starting version `2.x` (at the moment we use `2.2.0`).

Reply:
@sertony As far as I understand, the main task of Reanimated is to optimize native animation, but on the web this is common JS. Then why rely on reanimated in web when we can use stable React Native animations? 

Also I don't really understand how to implement this through reanimated, after all, I don't have much experience with him. Therefore, in order not to waste time, I used PanResponder from RN.